pkg/clients/aws/route53.go

```diff
@@ -57,6 +60,9 @@ const (
 	configMapSTSJumpRoleField   = "sts-jump-role"
 )
 
+var fedramp = os.Getenv(fedrampEnvVariable) == "true"
```

Comment:
I think os.LookupEnv() would be a little safer here as it handles empty/nil values better.
https://pkg.go.dev/os#LookupEnv